modules/sdk-coin-avaxp/src/lib/importInCTxBuilder.ts

```diff
@@ -0,0 +1,275 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BuildTransactionError, NotSupported, TransactionType } from '@bitgo/sdk-core';
+import {
+  EVMConstants,
+  Tx as EMVTx,
+  ImportTx,
+  UnsignedTx,
+  SECPTransferInput,
+  SelectCredentialClass,
+  TransferableInput,
+  EVMOutput,
+  AmountInput,
+} from 'avalanche/dist/apis/evm';
+import { costImportTx } from 'avalanche/dist/utils';
+import utils from './utils';
+import { BN, Buffer as BufferAvax } from 'avalanche';
+import { Transaction } from './transaction';
+import { Credential } from 'avalanche/dist/common';
+import { Tx, BaseTx } from './iface';
+import { TransactionBuilder } from './transactionBuilder';
+
+export class ImportInCTxBuilder extends TransactionBuilder {
+  private static _targetChainId: BufferAvax = utils.cb58Decode('yH8D7ThNJkxmtkuv2jgBa4P1Rn3Qpr4pPr7QYNfcdoS6k6HWp');
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  fee(value: string | number): this {
+    const bnValue = new BN(value);
+    this.validateFee(bnValue);
+    this.transaction._fee = bnValue;
+    return this;
+  }
+
+  to(cAddress: string): this {
+    const toAddress = BufferAvax.from(cAddress.slice(2), 'hex');
+    this.transaction._to = toAddress;
+    return this;
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.Import;
+  }
+
+  initBuilder(tx: Tx): this {
+    const baseTx: BaseTx = tx.getUnsignedTx().getTransaction();
+    if (
+      baseTx.getNetworkID() !== this.transaction._networkID // || !baseTx.getBlockchainID().equals(this._transaction._blockchainID)
+    ) {
+      throw new Error('Network or blockchain is not equals');
+    }
+
+    if (!this.verifyTxType(baseTx)) {
+      throw new NotSupported('Transaction cannot be parsed or has an unsupported transaction type');
+    }
+
+    // good assumption: addresses that unlock the outputs, will also be used to sign the transaction
```

Comment:
hmmm, will this be true all the time? 
What if it's not true.... 
Also have we filtered out non-multisig utxos?